cppcache/integration-test/testPoolFactory.cpp

```diff
@@ -30,9 +30,8 @@ BEGIN_TEST(POOLFACTORY)
   auto cacheFactory = CacheFactory();
   auto cache = cacheFactory.create();
   auto poolFactory = cache.getPoolManager().createFactory();
-  ASSERT(poolFactory != nullptr, "poolFactory was a nullptr");
-  auto& testPoolFactory = poolFactory->setSubscriptionEnabled(true);
-  bool test = poolFactory.get() == std::addressof<PoolFactory>(testPoolFactory);
+  auto& testPoolFactory = poolFactory.setSubscriptionEnabled(true);
+  bool test = &poolFactory == std::addressof<PoolFactory>(testPoolFactory);
```

Comment:
Do you think this test is really valuable anymore? Maybe PoolFactory should implement `operator==()` and we can compare them this way?

Reply:
I had wondered this as well.  My interpretation is that this test is trying to assert that the fluent interface is returning the exact same object which is probably not how one would usually expect `operator==()` to work.  I am fine deleting this test if you agree that it has minimal value.

Reply:
Oh.. I misread what it was doing. Yes leave it.
👍